lib/perl/Genome/Model/Tools/FastTier/TierBitmasks.pm

```diff
@@ -481,3 +318,203 @@ sub bases_covered {
     }
     return $total;
 }
+
+sub chromosome_name_list {
```

Comment:
Could this be replaced with a call to `$self->reference_sequence_build->chromosome_array_ref`?